satellite/plotter.py

```diff
@@ -0,0 +1,134 @@
+import pandas as pd
+import matplotlib.pyplot as plt
```

Comment:
Does matplotlib is a sub dependency of other dependency ? Because it is not listed on the requirements.txt